app/menu.js

```diff
@@ -67,40 +58,6 @@ function createMenu(opts) {
     }
   ]
 
-  const checkForUpdates = {
-    label: 'Check for Updates',
-    async click(item, focusedWindow) {
-      const api =
-        'https://api.github.com/repos/egoist/devdocs-app/releases/latest'
-      const latest = await axios.get(api).then(res => res.data)
-
-      if (semverCompare(latest.tag_name.slice(1), pkg.version) === 1) {
-        dialog.showMessageBox(
-          focusedWindow,
-          {
-            type: 'info',
-            message: 'New updates!',
-            detail: `A new release (${latest.tag_name}) is available, view more details?`,
-            buttons: ['OK', 'Cancel'],
-            defaultId: 0
-          },
-          selected => {
-            if (selected === 0) {
-              shell.openExternal(
-                'https://github.com/egoist/devdocs-app/releases/latest'
-              )
-            }
-          }
-        )
-      } else {
-        dialog.showMessageBox(focusedWindow, {
-          message: 'No updates!',
-          detail: `v${pkg.version} is already the latest version.`
-        })
-      }
-    }
-  }
-
```

Comment:
I can add it yeah! But I would prefer to make it in a separate PR, I'm already working on it actually. Both that and the everyday check which existed before.

This PR concentrates on the automatic builds and the download and install of the updates. The other PR will focus on checking for updates and feedback (ie: problems with download and such)